crates/bevy_pbr/src/fog.rs

```diff
@@ -462,7 +462,7 @@ impl FogFalloff {
     /// - <https://en.wikipedia.org/wiki/Visibility>
     /// - <https://www.biral.com/wp-content/uploads/2015/02/Introduction_to_visibility-v2-2.pdf>
     pub fn koschmieder(v: f32, c_t: f32) -> f32 {
-        -c_t.ln() / v
+        ln(-c_t) / v
```

Comment:
Pretty sure this is wrong, `.ln()` precedes negation.
The doc comment agrees that was the intention.
```suggestion
        -ln(c_t) / v
```